File: submissions/reservoir-coders/Challenge5.cs
static void f(string s)
{
    var t = " " + s + " ";
    s = t;
    do
    {
        t = s;
        s = s.Replace(@"/|\", "s");
        s = s.Replace("/ ", "__");
        s = s.Replace("/|", "//");
        s = s.Replace(@" \", "__");
        s = s.Replace(@"|\", @"\\");
    } while (t != s);
    s = s.Replace("s", @"/|\");

    Console.WriteLine(s.Trim());
}

File: submissions/reservoir-coders/Challenge5.go
func r(s,p,r string) string{return strings.Replace(s,p,r,-1)}
func f(s string){t:=" "+s+" ";s=t;for{t=s;s=r(s,"/|\\","s");s=r(s,"/ ","__");s=r(s,"/|","//");s=r(s," \\","__");s=r(s,"|\\","\\\\");if t==s{break}}
s=r(s,"s","/|\\");fmt.Print(strings.Trim(s," "));}